Makefile

```diff
@@ -24,6 +23,8 @@ update-hub:
 #
 # Test tiers.
 #
+# Set HUB_BASE_URL to point to Konveyor installation, e.g. export HUB_BASE_URL="http://192.168.39.16/hub"
```

Comment:
Could you please add this also to README, as required?

Reply:
Added.